_docs/tasks/installing-istio.md

```diff
@@ -36,7 +36,9 @@ This page shows how to install and configure Istio in a Kubernetes cluster.
 * If you previously installed Istio on this cluster, please uninstall first by following the
   [uninstalling]({{home}}/docs/tasks/installing-istio.html#uninstalling) steps at the end of this page.
 
-## Installation steps
+You can use the [Istio Helm chart](https://github.com/kubernetes/charts/tree/master/incubator/istio) to install, or follow the steps below for manual installation:
+
+## Manual installation steps
```

Comment:
I would move the paragraph above inside of the # Installation steps header, and leave the header name unchanged.

There should be steps in the Uninstalling section regarding Helm charts too.